lean4/app/ReoptVCG.lean

```diff
@@ -0,0 +1,35 @@
+import ReoptVCG.Annotations
+
+
+namespace ReoptVCG
+
+inductive VerificationMode
+| defaultMode : VerificationMode
+| exportMode : String → VerificationMode
+| runSolverMode : String → List String → VerificationMode
+
+def isDefaultMode : VerificationMode → Bool
+| VerificationMode.defaultMode => true
+| _ => false
+
+
+-- Like VCGArgs in Main but with all mandatory fields no longer as Options.
+structure VCGConfig :=
+  (annFile : String)
+  (mode : VerificationMode)
+  (verbose : Bool)
+          
+
+def runVCG (cfg : VCGConfig) : IO UInt32 :=
+do annContents ← IO.FS.readFile cfg.annFile;
```

Comment:
Lean4 uses hanging do, which I'm partial to myself.  I'd like to keep consistency with Lean.